### PR TITLE
Bugs/Profile/CLK32-1226-Fixbug-Twofactor

### PR DESCRIPTION
**Pull request by Minhdn1**
## Prerequisite
- [x] Self-review :monocle_face:
- [x] No Crashes ☠️
---

### Root cause
- // Explain what you have done

### Solution
- // Explain how you fix it

### Impact
- // List of affected components/modules

### Screenshots (Optional)
- // Capture affected screens

### CODE REVIEW

File: ClearKeep.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2366,7 +2366,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = com.telred.clearkeep.dev;
 				PRODUCT_NAME = "$(IOS_APP_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "dev-clk-22jul2022";
+				PROVISIONING_PROFILE_SPECIFIER = "clk-ios-dev";
```
Comment (vmo-klinh) [inline ClearKeep.xcodeproj/project.pbxproj]: discard

File: ClearKeep.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2717,7 +2717,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = "$(IOS_NOTIFICATION_EXTENSION_BUNDLE_ID)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "dev-noti-extension-clk-22jul2022";
+				PROVISIONING_PROFILE_SPECIFIER = "dev-notificationextension";
```
Comment (vmo-klinh) [inline ClearKeep.xcodeproj/project.pbxproj]: discard

**Review by vmo-klinh — CHANGES_REQUESTED** [review]

File: ClearKeep.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Minhdn1) [reply ClearKeep.xcodeproj/project.pbxproj]: fix complete

File: ClearKeep.xcodeproj/project.pbxproj (hunk 2 above)
Comment (Minhdn1) [reply ClearKeep.xcodeproj/project.pbxproj]: fix complete

**Review by vmo-klinh — APPROVED** [review]

**Pull request merged**